dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java

```diff
@@ -215,11 +216,11 @@ public Response toolsIdVersionsVersionIdTypeTestsPost(@ApiParam(hidden = true) @
     @ApiOperation(value = ExecutionMetricsPost.SUMMARY, notes = ExecutionMetricsPost.DESCRIPTION, authorizations = {
         @Authorization(value = JWT_SECURITY_DEFINITION_NAME)})
     @ApiResponses(value = {
-        @ApiResponse(code = HttpStatus.SC_NO_CONTENT, message = ExecutionMetricsPost.OK_RESPONSE),
+        @ApiResponse(code = HttpStatus.SC_MULTI_STATUS, message = ExecutionMetricsPost.MULTI_STATUS_RESPONSE),
```

Comment:
Highlighting that this is a multi-status response. The endpoint returns a list of responses, each one corresponding to an execution and if it was successfully submitted to Dockstore. Since each execution is submitted as individual files to S3 instead of one file, it's possible for one execution to be submitted successfully and another to fail for whatever reason